metadata.json

```diff
@@ -79,18 +72,10 @@
         "9"
       ]
     },
-    {
-      "operatingsystem": "Scientific",
-      "operatingsystemrelease": [
-        "7",
-        "8"
-      ]
-    },
     {
       "operatingsystem": "CentOS",
       "operatingsystemrelease": [
         "7",
```

Comment:
CentOS 7 is also EoL: https://voxpupuli.org/blog/2024/05/08/CentOS-7-CentOS-Stream-8-deprecation/

Reply:
Yes I'v planned to remove EL7 code and OS in a separe PR, need to remove a lot of unused logic from manifest and spec